.github/workflows/run-end-to-end.yml

```
@@ -158,6 +158,11 @@ jobs:
       run: ./run.sh TRACING_CONFIG_NONDEFAULT
       env:
         DD_API_KEY: ${{ secrets.DD_API_KEY }}
+    - name: Run TRACING_CONFIG_NONDEFAULT_2 scenario
+      if: always() && steps.build.outcome == 'success' && contains(inputs.scenarios, '"TRACING_CONFIG_NONDEFAULT_2"')
+      run: ./run.sh TRACING_CONFIG_NONDEFAULT_2
+       env:
+        DD_API_KEY: ${{ secrets.DD_API_KEY }}
```

you got a small typo that prevent the CI to run : 

```suggestion
      run: ./run.sh TRACING_CONFIG_NONDEFAULT_2
      env:
        DD_API_KEY: ${{ secrets.DD_API_KEY }}
```